ldm/invoke/CLI.py

```diff
@@ -213,7 +214,8 @@ def main_loop(gen, opt):
         set_default_output_dir(opt,completer)
 
         # try to relativize pathnames
-        for attr in ('init_img','init_mask','init_color','embedding_path'):
+        #        for attr in ('init_img','init_mask','init_color','embedding_path'):
```

Comment:
either `embedding_path` should be put back or the reference to it below should be deleted

Reply:
The reference is now deleted. This section was relativising paths to the output directory, which makes sense for images but not for the embedding path. I had commented the reference to embedding path just in case there was some reason for this that would show up during testing, but no, just a past mistake.